docs/en/operations/settings/query_complexity.md

```diff
@@ -73,6 +73,17 @@ Maximum number of unique keys received from aggregation. This setting lets you l
 What to do when the number of unique keys for aggregation exceeds the limit: 'throw', 'break', or 'any'. By default, throw.
 Using the 'any' value lets you run an approximation of GROUP BY. The quality of this approximation depends on the statistical nature of the data.
 
+## max_bytes_before_external_group_by {#settings-max_bytes_before_external_group_by}
+
+Enables of disables execution of `GROUP BY` clause in external memory. For more details on how it works, see [GROUP BY in external memory](../../#select-group-by-in-external-memory).
```

Comment:
* Enables o**r**
* For **more** details -> For explanation of
* Shouldn't the link lead to some md file instead of just ../..?